safe-global/safe-wallet-web
# Feat: Native Swaps

## What it solves

Resolves #3585 

## How this PR fixes it

Adds a native swap widget to a new page and shows decoded info for swap orders.

## How to test it

1. Open a Safe
2. Go to the dashboard
3. Observe a Swap button in the top right
4. Observe a Swap button in the Sidebar
5. Go to the Assets page
6. Observe a Swap button next to each asset

## Screenshots

## Checklist
